Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -59,46 +68,32 @@ - (BOOL)application:(UIApplication *)application
   [MSAppCenter setLogLevel:MSLogLevelVerbose];
 
   // Start App Center SDK.
-  BOOL useOneCollector = [[NSUserDefaults standardUserDefaults]
-      boolForKey:kMSOneCollectorEnabledKey];
-  long startTarget =
+  [MSAppCenter startFromLibraryWithServices:@[ [MSAnalytics class] ]];
```

Comment:
Should be moved to transmission tab so that we can check Analytics mode appears disabled when we go to Analytics tab directly, like in the Android and React Native apps.

Reply:
Can you do another PR for that but leave vsts task open? I will merge this increment.

Reply:
#1117